agents/a2a_framework.py

```
@@ -2,260 +2,254 @@
+
 class A2AMessage:
     """Standard message format for agent communication"""
-    
-    def __init__(self, 
-                 sender: str,
-                 recipient: str,
-                 message_type: str,
-                 content: Dict,
-                 conversation_id: str = None):
+
+    def __init__(
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)